configure.ac

```diff
@@ -647,6 +656,13 @@ AS_IF([test "x$enable_emscripten" = "xyes"],[
        CXXFLAGS="$CXXFLAGS -Wno-warn-absolute-paths -I$with_sdl2/include"
     ])
   ])
+  AS_IF([test "x$with_sdl_net" != "xno"],[
+    AS_IF([test "x$with_sdl_net" = "xyes"],
+      [CXXFLAGS="$CXXFLAGS -s USE_SDL_NET=2"],
+      [LIBS="$LIBS -L$with_sdl_net/build/.libs/ -lSDL2_net"
```

Comment:
Why `-L$with_sdl_net/build/.libs/`? The SDL_net library provided by Emscripten should link with only `-lSDL2_net` when using  `em++ -s USE_SDL=2 -s USE_SDL_NET=2`